Add FDB logs instructions

vendethiel opened the pull request.

vendethiel committed:
Add FDB logs instructions

jnthn commented on foundationdb/README.md:
````diff
@@ -53,6 +53,36 @@ FoundationDB does not include any events.
 
+## Logging
+
+1. FoundationDB has XML logs by default, to get JSON logs, add a `trace_format` key to your `foundationdb.conf`'s `fdbserver` section:
+
+```
+[fdbserver]
+...
+logdir = /var/log/foundationdb
+trace_format = json
+```
+
````
Maybe mention that one will need to restart the server after that?

jnthn commented on foundationdb/README.md:
````diff
@@ -53,6 +53,36 @@ FoundationDB does not include any events.
+```
+
+3. Add this configuration block to your foundationdb.d/conf.yaml file to start collecting your FoundationDB logs:
+The `path` here is the `logdir` in your configuration file. Make sure the datadog agent has the privileges required to list the directory and read its files.
````
Maybe the explanation bit wants to go after the example, given the `:`?

jnthn reviewed (commented):
The file `datadog_checks/foundationdb/data/conf.yaml.example` should also be updated to have a (commented out) example of the logs collection, so folks can uncomment and tweak.

vendethiel committed:
Update conf.yaml.exampe, slight changes to README

jnthn committed:
Comment out and document logs entry

We should not enable this by default, as it can send a really quite
large amount of data.
